base/docs/Docs.jl

```diff
@@ -12,7 +12,7 @@ const modules = Module[]
 
 const META′ = :__META__
 
-meta(mod) = mod.(META′)
+@eval meta(mod) = mod.$META′
```

Comment:
@tkelman 

> No not really, I have no clue what to look for.

Not being able to see anything obviously wrong from reading the code, the only think I can suggest is to print the following field of [`MCE`](https://github.com/llvm-mirror/llvm/blob/a93239b7c6f0d78cb8836768c3ffbc39fb15b79f/lib/Target/X86/X86CodeEmitter.cpp#L47) in [the loop](https://github.com/llvm-mirror/llvm/blob/a93239b7c6f0d78cb8836768c3ffbc39fb15b79f/lib/Target/X86/X86CodeEmitter.cpp#L143) you are stucked in [`runOnMachineFunction`](https://github.com/llvm-mirror/llvm/blob/a93239b7c6f0d78cb8836768c3ffbc39fb15b79f/lib/Target/X86/X86CodeEmitter.cpp#L134) before **and** after [calling `MCE.startFunction(MF)`](https://github.com/llvm-mirror/llvm/blob/a93239b7c6f0d78cb8836768c3ffbc39fb15b79f/lib/Target/X86/X86CodeEmitter.cpp#L145) as well as before [calling `MCE.finishFunction(MF)`](https://github.com/llvm-mirror/llvm/blob/a93239b7c6f0d78cb8836768c3ffbc39fb15b79f/lib/Target/X86/X86CodeEmitter.cpp#L159) (so three points in the loop).

```c++
ActualSize;
SizeEstimate;
BufferBegin;
SavedBufferBegin;
BufferEnd;
SavedBufferEnd;
CurBufferPtr;
SavedCurBufferPtr;
```

I don't remember how `gdb` prints `C++` classes by default and it might easier to print out the whole class.

Watching the address of these might also be helpful but that might generate too much noise before we know more details.

If no one else has better idea, this should at least help us to narrow down the problem.

> Or you could probably head in and do some work on the dual-boot laptop on @andreasnoack's desk during the week.

I'm afraid not the coming weeks. (Weekends works for me though....)